Tribler/LICENSE.txt

```diff
@@ -1,679 +0,0 @@
-    Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
-
-
-    Delft University of Technology, 
-    Faculty of Electrical Engineering, Mathematics and Computer Science, 
-    Mekelweg 4, 
-    2628 CD Delft, 
-    The Netherlands
-
-    Kungliga Tekniska Högskolan (The Royal Institute of Technology), 
-    KTH/ICT/ECS/TSLab
-    Electrum 229
-    164 40 Kista
-    Sweden
-
-------------------------------------------------------------------------------
-
-    QLectives Dispersy and quality communities library contribution.
-
-    The research leading to this library has received funding from the European
-    Community's Seventh Framework Programme in the QLectives project under grant
-    agreement no 231200.
-
-    The following library modules are Copyright (c) 2008-2013,
-    Delft University of Technology;
-    All rights reserved:
-
-       Tribler/dispersy/*
-       Tribler/community/*
-
-    All library modules are free software, unless stated otherwise; you can 
-    redistribute them and/or modify them under the terms of the GNU Lesser 
-    General Public License as published by the Free Software Foundation; in 
-    particular, version 2.1 of the License.
-
-    This library is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-    Lesser General Public License for more details.
-
-    You should have received a copy of the GNU Lesser General Public
-    License along with this library; if not, write to the Free Software
-    Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
-
-    Delft University of Technology, 
-    Faculty of Electrical Engineering, Mathematics and Computer Science, 
-    Mekelweg 4, 
-    2628 CD Delft, 
-    The Netherlands
-
--------------------------------------------------------------------------------
-
-Unless otherwise noted, all files written by Bram Cohen, John Hoffman, Petru 
-Paler, Uoti Urpala, Ross Cohen, Tim Tucker, Choopan RATTANAPOKA, Yejun Yang,
-Myers Carpenter, Bill Bumgarner, Henry 'Pi' James, Loring Holden, 
-Dustin Pate ("noirsoldats@codemeu.com"), kratoak5, Roee Shlomo, Greg Fleming, 
-N. Goldmann ("Pir4nhaX,www.clanyakuza.com"), and Michel Hartmann is released
-under the MIT license, exceptions contain licensing information in them.
-
-Copyright (C) 2001-2002 Bram Cohen
-All rights reserved:
-
-     Tribler/Core/Utilities/btformats.py
-     Tribler/Core/Utilities/parseargs.py
-     Tribler/Core/Utilities/bencode.py
-     Tribler/Core/Utilities/convert.py
-     Tribler/Core/Utilities/bitfield.py
-     Tribler/Core/APIImplementation/maketorrent.py
-     Tribler/Core/DecentralizedTracking/ut_pex.py
-     Tribler/Core/RawServer/SocketHandler.py
-     Tribler/Core/RawServer/selectpoll.py
-     Tribler/Core/RawServer/RawServer.py
-     Tribler/Core/defaults.py
-
```

Comment:
So it has been checked that all these files are no longer present. Tribler project has replaced them all, thus holds full copyright on these original works.